PD-1405 / 24.10 / PD-1405-the-static-ip-settings-from-console-are-inaccurate (#3034)
truenas/documentation: content/SCALEUIReference/SystemSettings/Services/NFSServiceScreen.md

{{< truetable >}}
| Setting | Description |
|---------|-------------|

> | **Bind IP Addresses** | Select IP addresses to listen to for NFS requests. Leave empty for NFS to listen to all available addresses. You must configure static IPs, also referred to as aliases, on the interface to appear on the dropdown list. |

Reviewer: ```suggestion
| **Bind IP Addresses** | Select IP addresses to listen to for NFS requests. Leave empty for NFS to listen to all available addresses. Static IPs configured show on the interface to appear on the dropdown list. |
```
